next.config.js

```diff
@@ -1,4 +1,13 @@
 module.exports = {
+  async redirects() {
+    return [
+      {
+        source: '/articles/category/:category/page/([0-1])',
+        destination: '/articles/category/:category',
```

Comment:
Would it be possible to have this in the new middleware to have all the related redirects in one place?